docker/docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: '3.1'
+services:
+  db2-ibm:
+      image: cwds/db2
+      ports:
+          - "50000:50000"
+      environment:
+          DB2INST1_PASSWORD: "db2inst1"
+          LICENSE: "accept"
+          DB2NAME: "cmsdb"
+      command: db2start
+
+  case-api:
+      image: mzemlyanoi/case-management-api
+      hostname: case-management-api
+      depends_on:
+        - db2-ibm
+      ports:
+        - "8080:8080"
+        - "9081:9081"
+      environment:
+         SHIRO_CONFIG_PATH: "/opt/case-management-api/config/shiro.ini"
```

Comment:
I think we should put here all the env variables, but comment them. This is a way to overwrite yml config